cg/meta/orders/utils.py

```diff
@@ -30,6 +43,10 @@ def get_ticket_tags(order: Order, order_type: OrderType) -> list[str]:
         if contains_existing_data(order):
             tags.append("existing-data")
 
+    if isinstance(order, OrderWithSamples):
+        if contains_external_data(order.samples, status_db):
```

Comment:
We have a convention in cg, for functions with more than one parameter, we name them
```suggestion
        if contains_external_data(samples=order.samples, status_db=status_db):
```

Reply:
Ah, of course!